recipe/meta.yaml

```diff
@@ -77,6 +77,7 @@ outputs:
         - {{ cdt('libxdamage-devel') }}      # [linux]
         - {{ cdt('libxfixes-devel') }}       # [linux]
         - {{ cdt('libxxf86vm-devel') }}      # [linux]
+        - {{ cdt('libxshmfence-devel') }}    # [cdt_name=="cos7"]
```

Comment:
OK. Last issue is this `cdt_name` qualifier. Changing that to just `linux` makes it work. I've not seen `cdt_name` before, so I don't know the proper syntax. Can we just have `linux & (ppc64le | aarch64)` ?

Reply:
actually it looks like we have different things happening on aarch64 vs ppc64le. The CDTs are found on aarch64, but not GL. Its the other way around on ppc64le. Strange.

Reply:
OK.  One problem could be the mesa CDT package:

https://anaconda.org/conda-forge/mesa-libgl-cos7-aarch64/18.0.5/download/noarch/mesa-libgl-cos7-aarch64-18.0.5-h1341992_0.tar.bz2

It contains:
```
libGLX_mesa.so.0
libGLX_mesa.so.0.0.0
```
It *should* contain `libGL.so`. The recipe looks right, this actually comes from the CentOS package itself. Very strange.

Reply:
More exploration: This is a change between the 17.* and 18.* of the mesa-libGL package.

Reply:
libGL.so has moved to `libglvnd-glx`. So we'll need a new CDT for that one for aarch64. The ppc64le libGL-mesa CDT comes from defaults and is the 17.* version so it includes libGL.so. Messy.

Reply:
We should probably keep aarch64 and ppc64le the same, so getting the proper CDTs in for ppc64le is a good idea as well.

Reply:
nice hunting. Probably don't have too much time now, but I think I gave you access to my repo.

Reply:
OK. We needed a few more CDTs than I thought, but this one should *finally* do it:
https://github.com/conda-forge/staged-recipes/pull/10699